packages/fluentui/react-northstar/src/themes/teams/components/Tree/treeTitleStyles.ts

```diff
@@ -12,6 +14,52 @@ const treeTitleStyles: ComponentSlotStylesPrepared<TreeTitleStylesProps, TreeTit
     position: 'relative',
     ...getBorderFocusStyles({ variables: siteVariables }),
   }),
+
+  selectionIndicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: 'inline-block',
+    float: 'right',
+    boxShadow: 'unset',
+    width: pxToRem(16),
+    height: pxToRem(16),
+
+    borderColor: v.borderColor,
+    borderStyle: v.borderStyle,
+    borderRadius: v.borderRadius,
+    borderWidth: v.borderWidth,
+    color: v.indicatorColor,
+    margin: v.selectionIndicatorMargin,
+    padding: v.padding,
+    userSelect: 'none',
+
+    backgroundImage: checkboxIndicatorUrl(v.indicatorColor, v.background),
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+
+    ...(p.selected && {
+      borderColor: v.checkedBorderColor,
+      backgroundImage: checkboxIndicatorUrl(v.checkedIndicatorColor, v.checkedBackground),
+    }),
+
+    ...(p.disabled && {
+      background: v.disabledBackground,
+      borderColor: v.disabledBorderColor,
+    }),
+
+    ...(p.disabled &&
+      p.selected && {
+        color: v.disabledCheckedIndicatorColor,
+        borderColor: v.disabledBackgroundChecked,
+        backgroundImage: checkboxIndicatorUrl(v.disabledCheckedIndicatorColor, v.disabledBackgroundChecked),
+      }),
+  }),
+
+  customSelectionIndicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: 'inline-block',
```

Comment:
Move these styles in the selectionIndicator slot, with the condition that is now inside the `Box.create` function. See https://github.com/microsoft/fluentui/pull/12362/files#r405611881 Use the same condition for applying this styles as well as the `display: 'none'`